### ISSUE TITLE
Error in .releaserc  Yaml Style 

### ISSUE THREAD

**Issue by astephanh** [m1pl/semantic-release-helm#6]
Hi, 

i try to use the plugin with yaml Style Config. 

if i include a prepare Config, i can't set the path, because of the duplicate key. If i don't set the path, i'll get the error: path not found. So i only way i can use it, is by don't customize the prepare Stage and configure the Plugin globally. 

Example
```
prepare:
  - path: "@semantic-release/changelog"
  - path: "@semantic-release/exec"
    cmd: "helm-docs chart"
  - path: "semantic-release-helm"
    path: "chart"
  - path: "@semantic-release/git"
    message: "RELEASE: ${nextRelease.version}"
```

Error: 
```
YAMLSemanticError: YAML Error in /home/melvin/code/helm/helm-helloworld/.releaserc.yaml:
Map keys must be unique; "path" is repeated at line 110, column 5:

  - path: "semantic-release-helm"
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^…
```

**Comment by m1pl** [comment]
@astephanh Hmm, let me think about it. I could rename it, although this would be a breaking change. I'll get back to you later.

**Comment by m1pl** [comment]
@astephanh I released v2.0.0, should be available on npmjs shortly. The config parameter has been renamed to `chartPath`.
